src/server/scripts/Outland/zone_hellfire_peninsula.cpp

```diff
@@ -1039,6 +1041,259 @@ class npc_magister_aledis : public CreatureScript
     }
 };
 
+enum WatchCommanderLeonus
+{
+    SAY_COVER = 0,
+    EVENT_LEONUS_TALK = 1,
+    EVENT_INFERNAL_RAIN_ATTACK = 2,
+    EVENT_FEAR_CONTROLLER_CAST = 3,
+    EVENT_ACTIVE_FALSE = 4,
+    NPC_INFERNAL_RAIN = 18729,
+    SPELL_INFERNAL_RAIN = 33814,
+    NPC_FEAR_CONTROLLER = 19393
+};
+
+class npc_watch_commander_leonus : public CreatureScript
+{
+public:
+    npc_watch_commander_leonus() : CreatureScript("npc_watch_commander_leonus") { }
+
+    struct npc_watch_commander_leonusAI : public ScriptedAI
+    {
+        npc_watch_commander_leonusAI(Creature* creature) : ScriptedAI(creature)
+        {
+            _events.ScheduleEvent(EVENT_LEONUS_TALK, Minutes(2), Minutes(10));
+            _events.ScheduleEvent(EVENT_INFERNAL_RAIN_ATTACK, Minutes(2), Minutes(10));
+            _events.ScheduleEvent(EVENT_FEAR_CONTROLLER_CAST, Minutes(2), Minutes(10));
```

Comment:
Looks like we have support to literal chrono now
So can be changed to: 2m, 10m (event times)